dist/vespa.spec

```diff
@@ -171,7 +171,7 @@ Requires: vespa-xxhash >= 0.8.1
 Requires: xxhash-libs >= 0.8.1
 %endif
 %if 0%{?el8}
-Requires: vespa-openssl >= 3.1.7
+Requires: vespa-openssl >= 3.5.4
```

Comment:
Consider adding a `_vespa_openssl_version` macro to avoid having to repeat the version number later on.